app/views/products/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Products</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Description</th>
+      <th>Picture</th>
+      <th>Unit price</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @products.each do |product| %>
+      <tr>
+        <td><%= product.name %></td>
+        <td><%= product.description %></td>
+        <td><%= image_tag product.picture, size: "72" %></td>
```

Comment:
changed avatar to sit on left side. did not do anything else with style (distribution) as we can fix once pulled into master